src/registry/registry.ts

```diff
@@ -57,6 +58,16 @@ export interface FormsRegistryStoreState {
   postSubmissionActions: Array<PostSubmissionActionRegistration>;
 }
 
+export const controlTemplatesFactory = [
+  {
+    name: 'drug',
+    baseControlComponent: UISelectExtended,
+    datasource: {
+      id: 'drug',
+      config: {},
```

Comment:
You want to configure the drug class: see https://github.com/openmrs/openmrs-esm-form-entry/blob/c553a59f58f4e2aef3afd3fa3eaae0be4923c0df/src/app/form-data-source/form-data-source.service.ts#L144